tests/structured_tests/parse_error_missing_operand.toml

```diff
@@ -8,7 +8,7 @@ error = """
 error: expecting an operand for binary operator
   |
 1 | * | where response_ms *
-  |                       ^
+  |                         dangling binary operator
```

Comment:
lost the caret here?

Reply:
Yes, the annotate-snippets seemed to change behavior a bit and doesn't want to put the caret in when it's near the end of the snippet.  Need to fiddle with it some more.